### PR TITLE
Changes for NPM publish

### PR DESCRIPTION
**Pull request by cassidyking17**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
-  "name": "ecqm_module_generator",
-  "version": "1.0.0",
+  "name": "ecq_module_generator",
+  "version": "0.0.0",
```
Comment (okeefm) [inline package.json:3]: I would suggest the beta version should be `0.0.1`; I'm not sure how things will react to a `0.0.0` version number

**Review by okeefm — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
-  "name": "ecqm_module_generator",
-  "version": "1.0.0",
+  "name": "ecq_module_generator",
```
Comment (mgramigna) [inline package.json:2]:
```suggestion
  "name": "ecqm-module-generator",
```

**Review by mgramigna — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mgramigna) [reply package.json:3]: Yeah I was thinking this too. My plan was to have us do `npm version patch` which will automatically bump the version in package.json and create a 0.0.1 tag. Not sure if this is the best way. Technically we can change it and not use the npm version command and just create the tag manually for the first release. That's probably fine actually.

So in short, Cassidy you can change this to 0.0.1

**Review by mgramigna — APPROVED** [review]

**Review by okeefm — APPROVED** [review]
~Seems reasonable~ Looks good to me!

**Pull request merged**